packages/did/src/DidLinks/AccountLinks.chain.ts

```diff
@@ -125,27 +126,20 @@ function getUnprefixedSignature(
 }
 
 /**
- * Builds the parameters for an extrinsic to link the `account` to the `did` where the fees and deposit are covered by some third account.
- * This extrinsic must be authorized using the same full DID.
- * Note that in addition to the signing account and DID used here, the submitting account will also be able to dissolve the link via reclaiming its deposit!
+ * Generates the challenge that links a DID to an account.
+ * The account has to sign the raw challenge, while the DID will sign the extrinsic that contains the challenge and will
```

Comment:
```suggestion
 * The account has to sign the challenge, while the DID will sign the extrinsic that contains the challenge and will
```

What is a raw challenge?

Reply:
I think i wanted to stress the point that the account signs the challenge, while the did signs the challenge as well, but it's inside the extrinsic